addons/mail/models/discuss/discuss_channel_rtc_session.py

```diff
@@ -28,6 +33,11 @@ class MailRtcSession(models.Model):
          'There can only be one rtc session per channel member')
     ]
 
+    @api.model
+    def _get_asymmetric_keys(self):
```

Comment:
From the key generation method used (`generate_pem_keys`), that seems each call to `/mail/rtc/publickey` will regenerate a key pair. Am I missing something? Is it deterministic is some way?